src/app/app.component.html

```diff
@@ -1 +1,5 @@
-<app-navbar></app-navbar>
\ No newline at end of file
+<app-navbar></app-navbar>
+<app-jumbotron></app-jumbotron>
+<app-product></app-product>
+<app-video></app-video>
```

Comment:
replace all selector name prefix to tf. for ex: app-navbar it should be tf-navbar